operations/rates.md

```diff
@@ -287,183 +227,210 @@ The price in the response is dependent on the enterprise's [pricing](configurati
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `RateId` | string | required | Unique identifier of the [Rate](#rate) whose prices should be returned. |
-| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). |
-| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last time unit, in UTC timezone ISO 8601 format. The maximum size of time interval depends on the service's time unit: 367 hours if hours, 367 days if days, or 60 months if months. See [Time units](../concepts/time-units.md).  |
+| `FirstTimeUnitStartUtc` | string | optional | Start of the time interval, expressed as the timestamp for the start of the first [time unit](services.md#time-unit), in UTC timezone ISO 8601 format. |
+| `LastTimeUnitStartUtc` | string | optional | End of the time interval, expressed as the timestamp for the start of the last [time unit](services.md#time-unit), in UTC timezone ISO 8601 format. The maximum size of time interval depends on the service's [time unit](services.md#time-unit): 367 hours if hours, 367 days if days, or 24 months if months. |
+| `RateId` | string | required | Unique identifier of the [Rate](rates.md#rate) whose prices should be returned. |
+| `ProductId` | string | optional |  |
+| ~~`StartUtc`~~ | ~~string~~ | ~~optional~~ | **Deprecated!** |
+| ~~`EndUtc`~~ | ~~string~~ | ~~optional~~ | **Deprecated!** |
```

Comment:
suggestion: Hide this behind internal implementation.

Reply:
fixed [here](https://github.com/MewsSystems/gitbook-connector-api/pull/635/commits/303e1e9c613a36ef93074a7bf781ed4a0916aebf)